src/app/import-models/import-models.component.ts

```diff
@@ -69,40 +70,54 @@ export class ImportModelsComponent implements OnInit {
     private stateHandler: StateHandlerService,
     private broadcastSvc: BroadcastService,
     private stateService: StateService
-  ) { }
+  ) {}
 
   ngOnInit() {
     // tslint:disable-next-line: deprecation
-    this.importType = this.stateService.params.importType ? this.stateService.params.importType : 'dataModels';
+    this.importType = this.stateService.params.importType
```

Comment:
While modifying this area, could you also replace the deprecated `stateService.params` with `uiRouterGlobal.params` please?

Reply:
Updated to use UiRouterGlobals